include/tvm/relay/attrs/transform.h

```diff
@@ -605,6 +605,17 @@ struct StftAttrs : public tvm::AttrsNode<StftAttrs> {
   }
 };  // struct StftAttrs
 
+/*! \brief Attributes used in DFT operator */
+struct DFTAttrs : public tvm::AttrsNode<DFTAttrs> {
```

Comment:
Please document that this is 1D DFT along the last axis. 

We cannot use this op to support 2D DFT (PyTorch `fft2` op), correct?

Reply:
> Please document that this is 1D DFT along the last axis.

The description of the fact that calculations occur along the last axis is already given in the functions where the algorithm is implemented:
* `python/tvm/topi/signal.py`
* `python/tvm/topi/cuda/signal.py`
* `python/tvm/relay/op/transform.py`

> We cannot use this op to support 2D DFT (PyTorch fft2 op), correct?

We can use 1D DFT (which computes along the last axis) to calculate n-D DFT (and 2D DFT in particular). This can be done with the following pseudocode:
```Python
def swap_axes(a, axis1, axis2):
    permutation = list(range(len(shape(a))))
    permutation[axis1] = axis2
    permutation[axis2] = axis1
    return transpose(a, permutation)

def dftn(input_tensor, axes):
    output = input_tensor
    for axis in reversed(axes):
        output = swap_axes(output, axis, -1)
        output = DFT(output)
        output = swap_axes(output, axis, -1)
    return output
```
`axes` is a vector that contains the axes along which it is necessary to perform n-D Fourier transform (for 2D, the length of this vector must be 2).

The same approach is used in numpy: [here](https://github.com/numpy/numpy/blob/3b5eff00e49ed8ab1e6c9fa9345e51817a691b2f/numpy/fft/_pocketfft.py#L716) and [here](https://github.com/numpy/numpy/blob/3b5eff00e49ed8ab1e6c9fa9345e51817a691b2f/numpy/fft/_pocketfft.py#L701).